tabs/applications-setup/dwmtitus-setup.sh

```diff
@@ -295,11 +295,30 @@ setupDisplayManager() {
     
 }
 
+install_slstatus() {
+    printf "Do you want to install slstatus? (y/n): " # using printf instead of 'echo' to avoid newline, -n flag for 'echo' is not supported in POSIX
```

Comment:
```suggestion
    printf "Do you want to install slstatus? (y/N): " # using printf instead of 'echo' to avoid newline, -n flag for 'echo' is not supported in POSIX
```
Needs a default option, I recommend "N"

Reply:
Done